lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -208,6 +203,7 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
         state
       ) do
     state = ensure_evaluator(state, container_ref)
+    opts = Keyword.put_new(opts, :notify_to, self())
```

Comment:
We always want to get the message.

```suggestion
    opts = Keyword.put(opts, :notify_to, self())
```